internal/js/modules/k6/browser/browser/frame_mapping.go

```diff
@@ -448,6 +448,31 @@ func mapFrame(vu moduleVU, f *common.Frame) mapping {
 				return nil, f.WaitForURL(val, popts, jsRegexChecker)
 			}), nil
 		},
+		"waitForResponse": func(url sobek.Value, opts sobek.Value) (*sobek.Promise, error) {
```

Comment:
We don't want to support `frame.waitForResponse` just `page.waitForResponse`, to match Playwright. At least for now, until there is a definite use case for `frame.waitForResponse`.